README.md

```diff
@@ -3,8 +3,9 @@ Kids First Data Service
 ![Data Service](docs/dataservice.png)
 
 [![license](https://img.shields.io/github/license/kids-first/kf-api-dataservice.svg?style=for-the-badge)](https://github.com/kids-first/kf-api-dataservice/blob/master/LICENSE)
-[![CircleCI](https://img.shields.io/circleci/project/kids-first/kf-api-dataservice.svg?style=for-the-badge)](https://circleci.com/gh/kids-first/kf-api-dataservice/13?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)
 [![Docs](https://img.shields.io/readthedocs/pip.svg?style=for-the-badge)](http://kf-api-dataservice-qa.kids-first.io/)
+[![CircleCI](https://img.shields.io/circleci/project/kids-first/kf-api-dataservice.svg?style=for-the-badge)](https://circleci.com/gh/kids-first/kf-api-dataservice/13?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)
+[![Codacy grade](https://img.shields.io/codacy/grade/534528baa6d544ca9c2e2fbaad8d3a29.svg?style=for-the-badge)](https://app.codacy.com/app/dankolbman/kf-api-dataservice/dashboard)
```

Comment:
Haven't really used codacy too much before - is there a way we can put it under the general kids-first org, or does it have to be tied to a specific github account?

Reply:
I'm not sure. It didn't seem like it from the way the setup process was laid out.

Reply:
had to create an org under codacy